src/frontend/hooks/useSyncState.ts

```diff
@@ -56,8 +56,8 @@ function createSyncState(project: MapeoProjectApi) {
   const listeners = new Set<() => void>();
   let error: Error | undefined;
 
-  function onSyncState(state: SyncState) {
-    state = state;
+  function onSyncState(newState: SyncState) {
+    state = newState;
```

Comment:
this is more related to this rule, right? https://eslint.org/docs/latest/rules/no-shadow

not a big deal (we should eventually look into disabling that rule too)

Reply:
If I recall, it was violating both issues. (Not 100% sure, let me know if you want me to go back and check.)